RELEASE_NOTES.md

```diff
@@ -1,9 +1,10 @@
-# Upcoming
+https://github.com/Dargones/libraries# Upcoming
```

Comment:
```suggestion
# Upcoming
```

Rogue copy-and-paste perhaps? :)

Reply:
Thanks for catching that, @robin-aws.  Has been adjusted.

Reply:
Thanks for fixing them! If you resolve the latest conflicts from mainline this should be ready to merge.

Reply:
Done!